### ISSUE TITLE
S608: false positive

### ISSUE THREAD

**Issue by spaceone** [charliermarsh/ruff#2726]
```python
        print(dedent(
            r"""
            The local repository has been prepared. The repository can be updated using:

              univention-repository-update net

            The local host has been modified to use this local repository.  Other hosts
            must be re-configured by setting the Univention Configuration Registry (UCR)
            variable 'repository/online/server' to the FQDN of this host.

              ucr set repository/online/server="%(hostname)s.%(domainname)s"

            The setting is best set in a domain by defining UCR Policies, which
            set this variable on all hosts using this repository server. For example:

              udm policies/repositoryserver create \
                --position "cn=repository,cn=update,cn=policies,%(ldap/base)s" \
                --set name="%(hostname)s repository" \
                --set repositoryServer="%(hostname)s.%(domainname)s"
              udm container/dc modify \
                --dn "%(ldap/base)s" \
                --policy-reference "cn=%(hostname)s repository,cn=repository,cn=update,cn=policies,%(ldap/base)s"
            """ % configRegistry))

```

is detected as `S608`

**Comment by charliermarsh** [comment]
Unfortunately this is also flagged by Bandit so I consider it "up to spec".

**Comment by spaceone** [comment]
→ reported upstream: https://github.com/tylerwince/flake8-bandit/issues/43

**Comment by charliermarsh** [comment]
Thank you very much :)
